model/src/main/scala/aqua/model/func/ArgsCall.scala

```diff
@@ -0,0 +1,47 @@
+package aqua.model.func
+
+import aqua.model.{ValueModel, VarModel}
+import aqua.types.{ArrowType, DataType}
+import cats.syntax.functor._
+
+case class ArgsCall(args: List[ArgDef], call: List[Call.Arg]) {
```

Comment:
what it represents? why `args` is plural and `call` is singular? frustrating, given that the next is the `zipped` method